kaitaistruct.py

```diff
@@ -462,21 +590,21 @@ def __init__(self, expected, actual, io, src_path):
 
 class ValidationLessThanError(ValidationFailedError):
     """Signals validation failure: we required "actual" value to be
-    greater than or equal to "min", but it turned out that it's not.
+    greater than or equal to "minimal", but it turned out that it's not.
     """
-    def __init__(self, min, actual, io, src_path):
+    def __init__(self, minimal, actual, io, src_path):
         super(ValidationLessThanError, self).__init__("not in range, min %s, but got %s" % (repr(min), repr(actual)), io, src_path)
-        self.min = min
+        self.minimal = minimal
         self.actual = actual
 
 
 class ValidationGreaterThanError(ValidationFailedError):
     """Signals validation failure: we required "actual" value to be
-    less than or equal to "max", but it turned out that it's not.
+    less than or equal to "maximal", but it turned out that it's not.
     """
-    def __init__(self, max, actual, io, src_path):
+    def __init__(self, maximal, actual, io, src_path):
         super(ValidationGreaterThanError, self).__init__("not in range, max %s, but got %s" % (repr(max), repr(actual)), io, src_path)
-        self.max = max
+        self.maximal = maximal
```

Comment:
None of these changes are relevant to serialization. Please don't try to smuggle any unnecessary changes that only come from your personal opinions and preferences (and bring inconsistencies across runtime libraries, see https://github.com/kaitai-io/kaitai_struct_python_runtime/pull/48#discussion_r466304542) in a completely unrelated PR. For now, revert these changes and redo them in a separate PR if you want.

Oh, until now I haven't noticed that you opened this PR from the `master` branch of your fork repository, but it's still no excuse for doing such a big PR with many unrelated changes. Please remember this rule of thumb when making pull requests from your fork repository: **never commit directly on `master` of your fork** and **never open any pull requests from `master` of your fork**. Create a topic branch for every pull request you create. Read https://github.com/kaitai-io/kaitai_struct_doc/pull/38#issuecomment-653751666 for more info.

Using multiple topic branches allows you to create multiple pull requests at once, and thus you can keep the pull requests small and sticking to one particular topic. When you already opened this PR from `master`, keep it that way for now, but I still require that you stick to the topic of serialization which you have chosen.

Anyway, I see no reason here why rename `min` to `minimal` and `max` to `maximal`, all other runtime libraries use only `min` and `max`:

- [C++](https://github.com/kaitai-io/kaitai_struct_cpp_stl_runtime/blob/d020781b96ea1e2fe7e0ecf47ff3ae3c829ccd31/kaitai/exceptions.h#L99-L143)
- [C#](https://github.com/kaitai-io/kaitai_struct_csharp_runtime/blob/1b66a9a85f39c52728893400a23504844cc78e34/KaitaiStruct.cs#L125-L161)
- [Java](https://github.com/kaitai-io/kaitai_struct_java_runtime/blob/c342c6e836ddba03c9ec33a59034e209bb04a976/src/main/java/io/kaitai/struct/KaitaiStream.java#L617-L641)
- [JavaScript](https://github.com/kaitai-io/kaitai_struct_javascript_runtime/blob/1fd3006373547792589310bc0746556bb17a697e/KaitaiStream.js#L760-L780)
- PHP - [`ValidationLessThanError`](https://github.com/kaitai-io/kaitai_struct_php_runtime/blob/f96d48c5df5cde7162dc3458bb059e6b02e7eab3/lib/Kaitai/Struct/Error/ValidationLessThanError.php#L6-L15), [`ValidationGreaterThanError`](https://github.com/kaitai-io/kaitai_struct_php_runtime/blob/f96d48c5df5cde7162dc3458bb059e6b02e7eab3/lib/Kaitai/Struct/Error/ValidationGreaterThanError.php#L6-L15)
- [Ruby](https://github.com/kaitai-io/kaitai_struct_ruby_runtime/blob/4cefe6f5700b138c15fc0d7633946591d16b7211/lib/kaitai/struct/struct.rb#L578-L598)

So please remember these sentences that I wrote you in https://github.com/kaitai-io/kaitai_struct_python_runtime/pull/48#discussion_r466304542:

> If you believe that this established order doesn't make sense and you can come up with a much better one, feel free to open a [new issue](https://github.com/kaitai-io/kaitai_struct/issues/new) which will involve changing the order in all runtimes we have.

> It's not possible to scatter chaos and confusion by using different approaches across runtimes for no reason.

This applies for these `min` and `max` parameter names as well.

It might look that these particular names don't matter, but it's rather about the principle. I'll keep stopping any [salami tactics](https://en.wikipedia.org/wiki/Salami_tactics) involving any bad practices already in its infacy.